Cargo.toml

```diff
@@ -37,7 +37,8 @@ tracing-subscriber = "^0.3.17"
 tracing = "^0.1.37"
 himmelblau_unix_common = { path = "src/common" }
 kanidm_unix_common = { path = "src/glue" }
-msal = { version = "0.1.15" }
+#msal = { version = "0.1.15" }
+msal = { path = "../microsoft-authentication-library-for-rust" }
```

Comment:
Waiting on this msal MR: https://github.com/himmelblau-idm/microsoft-authentication-library-for-rust/pull/48